modules/ROOT/pages/index.adoc

```diff
@@ -1,5 +1,12 @@
 [[operations-manual]]
-= The Neo4j Operations Manual
+= Neo4j Operations
+:page-layout: docs-ndl
```

Comment:
I tried to build it locally, but it didn't render correctly. It complains with: "Page layout specified by page not found: docs-ndl (reverting to default layout)". Do you know what I can do to view it properly?